src/Components/VideoCard/HorizontalCard.jsx

```diff
@@ -1,33 +1,46 @@
 import "./HorizontalCard.css";
-import { useNavigate, useLocation} from "react-router-dom";
+import { useNavigate, useLocation } from "react-router-dom";
 import { useAuth } from "../../Contexts/AuthContext";
 import { useActions } from "../../Contexts/ActionsContext";
 import {
   addToLikedVideo,
   removeFromLikedVideo,
 } from "../../Services/LikeVideoServices";
 import { removeFromHistory } from "../../Services/HistoryServices";
+import { addToWatchLater, removeFromWatchLater } from "../../Services/WatchLaterServices";
 
 const HorizontalCard = ({ video }) => {
   const { _id, title, channel } = video;
   const navigate = useNavigate();
   const { actionsState, actionsDispatch } = useActions();
-  const { likesData } = actionsState;
+  const { likesData, watchLaterData } = actionsState;
   const { authState } = useAuth();
   const { token } = authState;
-  const { pathname } = useLocation()
+  const { pathname } = useLocation();
   return (
-    <div className="Horizontal-card" >
-      {pathname === "/history" && <i className="fas fa-times-circle delete-btn" onClick={()=> removeFromHistory(_id, token, actionsDispatch)}></i>}
+    <div className="Horizontal-card">
+      {pathname === "/history" && (
```

Comment:
you can name this better instead of `horizontal-card`  just make it `video-card` (iff it's not coming from component library)